internal/proto/third_party/google/api/visibility.proto

```diff
@@ -0,0 +1,111 @@
+// Copyright 2021 Google LLC
```

Comment:
This file is necessary unless we move to buf for generation (https://github.com/hashicorp/boundary/pull/1944), in which case it'll be removed together with the other vendored files.